package.json

```diff
@@ -33,7 +33,8 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.2.3",
     "rome": "11.0.0",
-    "socket.io": "^4.7.1",
+    "socket.io": "^4.7.4",
+    "socket.io-client": "^4.7.4",
```

Comment:
why you need this?

Reply:
The socket.io-client in order to test a client connection in the tests. The patch upgrade of socket.io nothing in particular, only for testing the latest version. Maybe it's more relevant the peer dependencies change

Reply:
@alemagio any thoughts on this?